### DISCUSSION TITLE
MongoDB Atlas Support

### DISCUSSION THREAD

**Discussion by sadstan** [cuigh/swirl#35] [Q&A]
Hi,
Can I use MongoDB Atlas instead of a local instance of MongoDB?
Ik its possible with an app like Petio (https://docs.petio.tv/configuration/first-time-setup#step-4)

**Comment by cuigh** [comment]
Sorry, I'm not familiar with MongoDB Atlas . As long as it is compatible with MongoDB’s protocol, there should be no problem in theory.
